simpleConditional/hasSpaceAtEnd.py

```diff
@@ -4,3 +4,7 @@
 #   hasSpaceAtEnd("")                  # return False
 #   hasSpaceAtEnd("Hello  World!")     # return False
 #   hasSpaceAtEnd("  Hello  World!  ") # return True
+from typing import List
+
+def hasSpaceAtEnd(s: str) -> bool:
+    return True if len(s) > 0 and s[-1] == ' ' else False
```

Comment:
Hum … okay, let

expression = len(s) > 0 and s[0] == ‘ ‘

Return True if expression is True. Otherwise if the expression is False return False.

Why you don’t just return the expression …

Reply:
Somehow I though this one should be “simple condition”, but turn out it is just “simple sequential”. I will move this to “simple sequential” later. For now, just KISS.

Reply:
ok.